### PR TITLE
Update bitcoin-qt.desktop

### PR DESCRIPTION
**Pull request by smatthewenglish**
I'd like to get the hang of how to make pull requests and contribute. there was no German there, but Turkish and French- many Bitcoiners are German- so- maybe it would be useful

### CODE REVIEW

**Comment by MarcoFalke** [comment]
This should probably be changed to just "Bitcoin Core GUI".

File: contrib/debian/bitcoin-qt.desktop [hunk 1]
```diff
@@ -3,6 +3,7 @@ Encoding=UTF-8
 Name=Bitcoin
```
Comment (MarcoFalke) [inline contrib/debian/bitcoin-qt.desktop]: "Bitcoin Core" or "Bitcoin Core GUI"

File: contrib/debian/bitcoin-qt.desktop [hunk 2]
```diff
@@ -3,6 +3,7 @@ Encoding=UTF-8
...
 Comment=Bitcoin P2P Cryptocurrency
```
Comment (MarcoFalke) [inline contrib/debian/bitcoin-qt.desktop]: I checked some programs and the default seems to be imperative style, so maybe change to:

Connect to the Bitcoin P2P Network

**Review by maflcko — CHANGES_REQUESTED** [review]

**Comment by s-matthew-english** [comment]
it's there some kind of- particular naming convention for the branch, if it should be merged? I guess I didn't do that correctly in this instance

**Comment by s-matthew-english** [comment]
I think I've made the change you requested, is it so?

File: contrib/debian/bitcoin-qt.desktop [hunk 3]
```diff
@@ -1,8 +1,9 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Bitcoin
-Comment=Bitcoin P2P Cryptocurrency
+Comment=Connect to the Bitcoin P2P Network
 Comment[fr]=Bitcoin, monnaie virtuelle cryptographique pair à pair
+Comment[de]=Bitcoin, eine virtuelle kryptographische peer-to-peer Währung
```
Comment (MarcoFalke) [inline contrib/debian/bitcoin-qt.desktop]: I guess this should match the English one. I.e.


"Verbinde mit dem Bitcoin peer-to-peer Netzwerk" (oder ähnlich)

Comment (s-matthew-english) [inline contrib/debian/bitcoin-qt.desktop]: ok cool, I've made the change.

File: contrib/debian/bitcoin-qt.desktop [hunk 4]
```diff
@@ -1,7 +1,8 @@
 [Desktop Entry]
 Encoding=UTF-8
 Name=Bitcoin
```
Comment (MarcoFalke) [inline contrib/debian/bitcoin-qt.desktop]: Please change this to "Bitcoin Core".

**Comment by s-matthew-english** [comment]
I think I've made a mistake which is, I made those changes directly in the window of GitHub, and they don't exist on my local repo, seems they're only in this patch here. Is there some way to incorporate those changes to my local repo so I can do the squashing procedure?

**Comment by laanwj** [comment]
> Is there some way to incorporate those changes to my local repo so I can do the squashing procedure?

```
git pull origin patch-4
```
(where `origin` is the name that you gave to your github remote)

**Comment by s-matthew-english** [comment]
so- origin/master? or...

**Comment by s-matthew-english** [comment]
now I need to issue the command `git rebase -i HEAD~n` isn't it? do you know how I can figure out what is `n`?

**Comment by fanquake** [comment]
@s-matthew-english `n` is the number of commits you've made.

**Comment by s-matthew-english** [comment]
my terminal tells me I'm ahead of origin/master by 5 commits, so then- is it 5? how can I check that?

**Comment by s-matthew-english** [comment]
hmm, I've been trying to squash it for a while now, did it work?

**Comment by s-matthew-english** [comment]
I think I squashed them, I followed this tutorial: https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request

**Comment by achow101** [comment]
@s-matthew-english didn't seem to work. Did you force push?

**Comment by s-matthew-english** [comment]
yeah :/ strange

**Comment by s-matthew-english** [comment]
do I force push it to upstream/master or origin/master?

**Comment by achow101** [comment]
it depends on which one points to your repo. It's probably origin/master, but you can try both. The worst that will happen is that you will get an error about not being able to push to one of those.

**Comment by s-matthew-english** [comment]
what about this: 

https://github.com/s-matthew-english/bitcoin/commit/238fbceeabe38b8e78635eb380ac10b2861197e5

**Comment by achow101** [comment]
Close. Do it for your patch-4 branch. Also, the commit message should be cut down to just one message that describes the changes simply.

**Comment by s-matthew-english** [comment]
hmm, is there a way to retroactively change to commit mesage? I think I've now done it for patch-4, did it work?

**Comment by achow101** [comment]
Nope. something very wrong happened. I think you pushed your whole master branch.

**Comment by s-matthew-english** [comment]
:/ shit- is there a way to... back it up to before I did that and then take another crack at it?

**Comment by achow101** [comment]
At this point you could just do it the "hard" way. Reset to Bitcoin master (`git reset --hard upstream/master` assuming upstream is to bitcoin/bitcoin) then make your final changes, commit, and force push.

**Comment by s-matthew-english** [comment]
on which branch 'patch-4'?

**Comment by achow101** [comment]
yes

**Comment by s-matthew-english** [comment]
:/ but when I run 'git merge-base patch-4 upstream/master' it says only
"noop"

**Comment by sipa** [comment]
Maybe this discussion can be held more effectively on IRC?

**Comment by achow101** [comment]
Don't do any merges. Just reset the branch like I told you above. You should end up doing something like this:

```
git checkout patch-4
git reset --hard upstream/master

# Do your changes again

git commit -a
git push -f origin
```

**Comment by s-matthew-english** [comment]
ok- I think I've just done it- did it work?

**Comment by achow101** [comment]
Almost. Somehow a change to src/crypto/hmac_sha256.cpp slipped in.

Just remove that change and then use `git commit -a --amend` to modify the existing commit.

**Comment by s-matthew-english** [comment]
ok- I think I've done it. is it so?

**Comment by achow101** [comment]
Yes, it is correct now.

**Comment by MarcoFalke** [comment]
Please do a last `git commit --ammend` and make sure to apply the second git commit message rule: http://chris.beams.io/posts/git-commit/#seven-rules

**Comment by s-matthew-english** [comment]
:D cool- thank you for helping me figure this out

**Comment by s-matthew-english** [comment]
how about that one?

**Comment by MarcoFalke** [comment]
utACK 164196b

**Comment by jonasschnelli** [comment]
utACK 164196b7c8aecb72dd20c8681bb24b81b18aecaa

**Comment by TheBlueMatt** [comment]
utACK 164196b7c8aecb72dd20c8681bb24b81b18aecaa

**Pull request merged**